examples/simpleReceive.ts

```diff
@@ -2,49 +2,38 @@ import { OnMessage, OnError, MessagingError, delay, Message, ReceiveMode, Namesp
 import * as dotenv from "dotenv";
 dotenv.config();
 
-const connectionString = "SERVICEBUS_CONNECTION_STRING";
-const entityPath = "QUEUE_NAME";
-const str = process.env[connectionString] || "";
-const path = process.env[entityPath] || "";
+const str = process.env["SERVICEBUS_CONNECTION_STRING"] || "";
+const path = process.env["QUEUE_NAME"] || "";
+
 console.log("str: ", str);
 console.log("path: ", path);
 
-let ns: Namespace;
 async function main(): Promise<void> {
-  ns = Namespace.createFromConnectionString(str);
+  const ns: Namespace = Namespace.createFromConnectionString(str);
```

Comment:
That is true. This will change. Shall be using the source code from here https://github.com/Azure/amqp-common-js/tree/master/lib/connectionConfig. Will be making some changes to ensure that entityPath is optional for namespace connectionConfig.